main/solution/post-deployment/config/environment-files/bin/mount_s3.sh

```diff
@@ -50,7 +50,8 @@ append_role_to_credentials() {
 
 # Use STS regional endpoint instead of global one. This allows external studies to connect with local interface endpoint
 # if it exists. Refer https://docs.aws.amazon.com/sdkref/latest/guide/setting-global-sts_regional_endpoints.html
-region=`curl http://169.254.169.254/latest/meta-data/placement/availability-zone/ | sed 's/.$//'`
+token=`curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600"`
```

Comment:
These commands worked to get regions for both versions? Did AL1 _require_ the token being passed now?

Reply:
Yep, it works for both version. AL1 didn't require the token, but it doesn't hurt to have the token. It makes the code simpler since there is less code branches.